public/assets/css/header.css

```
@@ -0,0 +1,27 @@
+body > header > nav > ul > li:first-child {
+    width: 64px;
+    height: 62px;
+    border: 0 solid #0098FF;
+    border-radius: 50%;
+    opacity: 0,5;
+    background-color: #0098FF;
```

you can replacethis line with
```suggestion
    background: rgb(128, 194, 233);
```